### ISSUE TITLE
@types does not work since v3

### ISSUE THREAD

**Issue by Pajn** [TypeStrong/ts-loader#661]
Since version 3 I can no longer use type definitions from `@types` packages.
For example I get this error from `@types/webpack`:
```
/Users/rasmus/Development/project/node_modules/@types/webpack/index.d.ts
[tsl] ERROR in /Users/rasmus/Development/project/node_modules/@types/webpack/index.d.ts(14,26)
      TS7016: Could not find a declaration file for module 'tapable'. '/Users/rasmus/Development/project/node_modules/tapable/lib/Tapable.js' implicitly has an 'any' type.
  Try `npm install @types/tapable` if it exists or add a new declaration (.d.ts) file containing `declare module 'tapable';`
```
`@types/tapable` is installed as a dependency of `@types/webpack` and its files looks good but I have tried with explicitly installing it as well with the same result.

It is not an error with the `@types/webpack` package because if I work around that one I get a similar error from `@types/uglify`. It seems like any `@types` package that import other modules is broken.

If I compile using tsc or ts-loader 2.3.7 there are no errors, neither does vscode report any errors.
My webpack configuration is super simple:
```js

          {
            test: /\.(ts|tsx)$/,
            include: paths.appSrc,
            use: [
              {
                loader: require.resolve('babel-loader'),
                options: {
                  cacheDirectory: true,
                },
              },
              require.resolve('ts-loader'),
            ],
          }
```

**Comment by Pajn** [comment]
Duplicate of #657

**Comment by johnnyreilly** [comment]
Just released v3.0.3 - check it out and let me know how it goes.

**Comment by johnnyreilly** [comment]
Great!
